File: basic Math/codeforces/JAVA/oop/Java lab/Lab4/Car.cpp
#include <iostream>
#include <string>
#include <vector>

class CarProperty {
public:
	std::string companyName;
	std::string modelName;
	std::string licenseNo;
	std::string fuelType;
	float speed;

	void SetProperty(std::string name, std::string model, std::string license, std::string fuel, float fSpeed);
};

void CarProperty::SetProperty(std::string name, std::string model, std::string license, std::string fuel, float fSpeed)
{
	companyName = name;
	modelName = model;
	licenseNo = license;
	fuelType = fuel;
	speed = fSpeed;
}

int main()
{
	std::vector<CarProperty>CarsInfo;

	CarProperty Car;
	Car.SetProperty("Toyota", "Supra", "skdfjs", "Gas", 120);	//Setter

	CarsInfo.push_back(Car)
;

	//CarsInfo.erase(CarsInfo.begin() + 2);
	return 0;
}

File: basic Math/codeforces/JAVA/oop/Java lab/Lab4/Car.java
public class Car {
    public String CarBrandName;
    public String CarModelName;
    public String CarNumber;
    public String CarPrice;
    public String CarFuleType;
    public float speed;
    
    public void setProperty(String brandName, String modelName, String licenseNumber, String fuelType, float carSpeed) {
        CarBrandName = brandName;
        CarModelName = modelName;
        CarNumber = licenseNumber;
        CarFuleType = fuelType;
        speed = carSpeed;
    }
    public static void main(String[] args){
        Car myCar = new Car();
        myCar.setProperty("Toyota", "Supra", "ABC123", "Gas", 180.0f);


        System.out.println(" " + myCar.CarBrandName);
        System.out.println(" " + myCar.CarModelName);
        System.out.println(" " + myCar.CarNumber);
        System.out.println(" " + myCar.CarFuleType);
        System.out.println(" " + myCar.speed );
    } 

    
    
}

